packages/dnb-eufemia/src/components/card/Card.tsx

```diff
@@ -118,11 +118,37 @@ function Card(props: Props) {
           )}
           {children}
         </Flex.Container>
-      </CardContext.Provider>
+      </Card.Provider>
     </Flex.Item>
   )
 }
 
 Card._supportsSpacingProps = true
 
+/**
+ * Provides the default context for the Card component's children.
+ * Can be used to tell children to act as if they are in a Card, even if they are not.
+ */
+Card.Provider = function CardProvider({
+  /**
+   * Defines if Card components should act as if it is nested inside another Card.
+   * Used to control side-margins outset behaviour.
+   *
+   * Defaults to existing context value.
+   */
+  hasParentCard,
```

Comment:
```suggestion
  suppressOutsetOfNestedCards,
```

I'd like to find a declarative name, so that a developer reading the prop can roughly understand what it does to the layout.

Any suggestions are welcome.

Reply:
I agree. The issue is that we aren't just disabling outset, we are also disabling responsive breakout. So it might be a bit misleading. You won't get the same behaviour as just setting `outset={false}` you also need `responsive={false}` 

Suggestions:

- forceCardsToRespectPadding
- disableOutsetAndBreakoutOfCards
- containCardsToContentArea

Reply:
The `responsive={false}` again will ensure it does not "break" out. So they both are within a mechanism that "breaks-out" from the layout. So could we maybe then use the term "breakout" instead of outset?

- `suppressBreakoutOfNestedCards`